.build/README.md

```diff
@@ -0,0 +1,134 @@
+Building and Testing with the helper sripts
+-------------------------------------------
+
+Information on building and testing beyond the use of ant.
+
+
+Code Checks and Lints
+---------------------
+
+Run in docker:
+
+    .build/docker/check-code.sh
+
+
+Run without docker:
+
+    .build/check-code.sh
+
+
+Run in docker with a specific jdk.
+The following applies to all build scripts.
+
+    .build/docker/check-code.sh 11
+
+
+Run in docker with a specific build path.
+This permits parallel builds off the same source path.
+The following applies to all build scripts.
+
+    build_dir=/tmp/cass_Mtu462n .build/docker/check-code.sh
+
+
+Building Artifacts (tarball and maven)
+-------------------------------------
+
+Build with docker:
+
+    .build/docker/build-artifacts.sh
+
+
+Build without docker:
+
+    .build/build-artifacts.sh
+
+
+Build in docker with a specific jdk:
+
+    .build/docker/build-artifacts.sh 11
+
+
+Building Debian and RedHat packages
+-----------------------------------
+
+Build with docker:
+
+    .build/docker/build-debian.sh
+    .build/docker/build-redhat.sh
+
+
+Build without docker:
+
+    .build/build-debian.sh
```

Comment:
@michaelsembwever these scripts are not in `.build` directory.

Reply:
fixed